pkg/controller/garbagecollector/garbagecollector.go

```diff
@@ -43,6 +44,8 @@ import (
 
 const ResourceResyncTime = 60 * time.Second
 
+const OrphanFinalizerID = "orphanFinalizer"
```

Comment:
Also please put it in a different package so apiserver doesn't have to import the garbage collector.